### DISCUSSION TITLE
Thetvdb - alternate order

### DISCUSSION THREAD

**Discussion by solipsist01** [pymedusa/Medusa#7916] [Ideas & Feature requests]
Hello.

https://www.thetvdb.com/series/la-casa-de-papel
If you look at la casa de papel on thevdb, there are two serperate airing schedules.

Aired order
Netflix aired order

It would be nice when you add a show, that you can choose between aired orders. Now it only uses original aired order, which in this case conflicts with various sources.

**Comment by p0psicles** [comment]
#282

**Comment by papampi** [comment]
+1

**Comment by p0psicles** [comment]
There is already a solution for that. Thexem.de can map numbering of tvdb to that of used in sources.

**Comment by papampi** [comment]
@p0psicles Can you please provide more info on how to do it?

**Comment by p0psicles** [comment]
Check thexem.de. they have an irc Channel Where you can contact them.

**Comment by p0psicles** [comment]
But Another problem is, that the show is releases in Both orders. So I don't know if there is a Real Solution to this

**Comment by p0psicles** [comment]
As you can, read here it's not a mapping issue. Episodes are re-editted. https://forums.sonarr.tv/t/manually-edit-seaons-and-number-of-episodes-casa-de-papel/18182 not much we can do

**Comment by papampi** [comment]
I used TinyMediaManager with imdb indexer to get nfo and rename them in my library, then added show by already exists in Medusa with incorrect info.
No worries kodi shows info from files and are correct.

**Comment by solipsist01** [comment]
@p0psicles Episodes are being editted by Netflix. But, thetvdb.com keeps track of **Both** airing schedules. Meaning, Original, and (editted) Netflix.

This is a new feature of Thetvdb.

In the past, the series La casa de papel has caused alot of uproar and drama on forums with season numberings and all. 
Thetvdb came up with a perfect solution for that.
The thread from Sonarr you are referring to, is from **before** this change on thetvdb

Check this screenshot.
![image](https://user-images.githubusercontent.com/16472221/80747030-0f98a680-8b23-11ea-8d98-682f2e77c1ff.png)

In my opinion, the best solution would be, that you can manually select which order you want to use, as both are present and maintained on thetvdb.

Am i missing something obvious then ? :) (ps, thexem.de doesn't have this show)

**Comment by p0psicles** [comment]
I have some other stuff to finish First. Will look into it after

**Comment by papampi** [comment]
Yup, they changed DVD order to Netflix

**Comment by p0psicles** [comment]
Ok more reason to carefully analyse this. As Medusa has had support for DVD order in the past. 

Also is this limited to Netflix, and just relabeld. Or also support other services like Amazon and Disney?

**Comment by papampi** [comment]
I haven't seen this for Amazon, Disney or others.
Only seen it for Netflix

**Comment by dausruddin** [comment]
@p0psicles you said Medusa has support for DVD in the past. Was it removed? 
I stumbled upon [this anime](https://www.thetvdb.com/series/re-zero-starting-life-in-another-world) and would like to use Director's Cut order.

**Comment by p0psicles** [comment]
I think more like lost when I refactored the displayShow Page. I got norified it was missing now.

**Comment by p0psicles** [comment]
https://forums.thetvdb.com/viewtopic.php?f=17&t=63756

The alternative order function is not available through tvdb api. They removed it. They're adding it through there apiv4. 

When that's finished, I'll add it.

**Reply by xbette** [reply]
Any news about this :D? Looks like V4 is released

**Reply by p0psicles** [reply]
Yeah. Basically, we're going to wait for the sunset date on tvdb v2 / v3 api. And then where going to look for a minimum way of still using tvdb.

Then for a solution on the alternate order. Where going to implement that for tmdb and tvmaze. (If possible)

**Comment by Rouzax** [comment]
Wanted to chime in for TMDB.
They also have the Episode Groups.
So for the same show `Money Heist` it would be helpful if I could select the Episode Groups I want to include.
https://www.themoviedb.org/tv/71446-la-casa-de-papel/episode_group/5eb730dfca7ec6001f7beb51
